src/schema.ts

```diff
@@ -1394,10 +1488,27 @@ export const typeDefs = gql`
     arenaCategories: [ArenaCategory!]!
     arenaCategory(categoryId: Int!, page: Int!): ArenaCategory
     arenaUser(username: String!): ArenaUser
+    arenaAllFlags(page: Int, pageSize: Int): PaginatedPosts!
```

Comment:
Er flags bare en liste med posts? Tjener vi noe på å separere de i cache, isåfall? Eventuelt, kan det gi oss noe ekstra kluss?

Reply:
Flags er en liste med poster som er flagget (som igjen har en liste med flagg).
Jeg vet ikke om jeg skjønner helt hva det betyr å separere de i cache, men når du sier det så vil flaggene bare væree tilgjengelig dersom man er admin så vi må kanskje gjøre noe for å cache disse individuelt om du er admin eller ikke? Gjør vi noe lignende noe sted?